www/front_src/src/Resources/Graph/Performance/ExportableGraphWithTimeline/index.tsx

```diff
@@ -166,6 +178,22 @@ const ExportablePerformanceGraphWithTimeline = ({
     ]);
   };
 
+  const getIsModalOpened = (value: boolean): void => {
```

Comment:
is this still needed?

Reply:
yes because from component graphAction which contains icon wrench we send the event (to the parent component ExportablePerformanceGraphWithTimeline) that the icon has been clicked so the modal of edit envelope size must be displayed